Source/WebCore/animation/BlendingKeyframes.cpp

```diff
@@ -343,7 +343,7 @@ void BlendingKeyframes::updatePropertiesMetadata(const StyleProperties& properti
                     m_usesAnchorFunctions = true;
             }
         } else if (auto* customPropertyValue = dynamicDowncast<CSSCustomPropertyValue>(cssValue)) {
-            if (customPropertyValue->isInherit())
+            if (auto keyword = customPropertyValue->tryCSSWideKeyword(); keyword && *keyword == CSSWideKeyword::Inherit)
```

Comment:
```suggestion
            if (customPropertyValue->tryCSSWideKeyword() == CSSWideKeyword::Inherit)
```
I'd think the above code would work given the semantics of `==` for `std::optional`.